src/pages/home/report/ReportActionItemSingle.tsx

```diff
@@ -77,21 +84,27 @@ function ReportActionItemSingle({
     report,
     iouReport,
     isHovered = false,
+    personalDetails,
+    policy,
 }: ReportActionItemSingleProps) {
     const theme = useTheme();
     const styles = useThemeStyles();
     const StyleUtils = useStyleUtils();
     const {translate} = useLocalize();
-    const personalDetails = usePersonalDetails();
-    const policy = usePolicy(report?.policyID);
+    const onyxPolicy = usePolicy(report?.policyID);
+    const reportPolicy = policy ?? onyxPolicy;
```

Comment:
> Why do we need to connect to Onyx here? shouldn't the policy always be available via policy prop?

@luacmartins the policy is now fully extracted


> I did a quick test tweaking the API to return all reportAction types and we can see that the actions are rendered,

Is it possible for me to perform such test? It would be easier to debug when the data exists.

Reply:
>@luacmartins the policy is now fully extracted

Thanks!

>Is it possible for me to perform such test? It would be easier to debug when the data exists.

Unfortunately not, this would require access to the dev API, which is not open source atm. I'll try to put up a PR to return that data already (if things in App don't break with the current logic). Hopefully that'll help.

Reply:
@luacmartins great, maybe add feature flag to the dev api with enabled set to false to return that data